### ISSUE TITLE
Learning rate decay for cat2dog dataset

### ISSUE THREAD

**Issue by korawat-tanwisuth** [HelenMao/MSGAN#19]
Hi @HelenMao,

python test.py --dataroot ./datasets/cat2dog --resume ./models/DRIT-Mode-Seeking/cat2dog/01999.pth --concat 0

Unlike summer2winter, I see that the model is ran for 2k epochs for cat2dog dataset. I am wondering if the number of epochs before we decay the learning rate is 600 or 1,000. I could not find the details of the exact hyperparameter settings.  
In the code, I saw in the comment that n_ep = 400 * d_iter. Does that also mean d_iter = 2000/400 = 5 in this case?
Thank you for your help!

**Comment by HelenMao** [comment]
Hi, in fact, in our CVPR setting we do not include any decay in the cat2dog dataset and only run 2000 epochs.
You can try decay=1000 if you like, but make sure you should make the training of the baseline model and MSGAN consistent if you want to compare these two models.
BTW, my experience when I am working on the Journal version is that when using decay learning rate the mode collapse of DRIT can be slightly mitigated. However, MSGAN still further improves the baseline model.

**Comment by korawat-tanwisuth** [comment]
Hi @HelenMao,

Thank you so much for your response! Do you still update the content discriminator for every 3 iteration for the cat2dog dataset?

**Comment by HelenMao** [comment]
Yes, we use the default setting of the DRIT configuration.

**Comment by korawat-tanwisuth** [comment]
Thanks!
